data/hai/hai reveal 4 middle.txt

```diff
@@ -0,0 +1,455 @@
+			choice
+				`	(Enter the building.)`
+			`	You're ushered to an elevator and given a room number: 3017A19 - the 301st floor, section 7A, room 19. It's amazing how large Earth's megabuildings have become. Some planets have less population than one of these buildings.`
+			`	Giti is even more excited in-person than she was by holocommunicator. She's already researched the Syndicate-like symbol, but needed more detail about the demonic one. You also show her the information about the microtransceivers.`
+			`	"This cat symbol was used by a clandestine organization that worked within the Syndicate over a half-century ago. They called themselves 'Base Principles.' They were anarcho-syndicalists infiltrating the Syndicate from within, supposedly advancing their positions through research findings gleaned from captured alien equipment, but nobody knows where they got it. The organization was eventually torn in two by a collection of regular anarchists and other unsavory characters looking to profit off the backdoor access to the Syndicate; this group kidnapped researchers from other institutes to expand their capabilities.`
+			`	"These others didn't cover their tracks well enough, the trail led back to the syndicalists that started it all in the first place, and Syndicate Internal Affairs put a stop to the organization with extreme prejudice. Most of the anarchist leaders were killed in battle or executed after capture, but the internal players simply... disappeared.`
+			`	"Officially, the Syndicate maintains that they wiped Base Principles out to the last individual, but evidence never proved it had been conclusively terminated.`
+			`	"For the demonic symbol, we'll have to go to another library in room 4196A21."`
+			choice
+				`	(Take the elevator to the 419th floor.)`
+			`	When you arrive, Giti tells you, "This is a religious materials archive. Give me a minute here..." Her voice trails off as she starts typing on a terminal. She walks back and forth between several bookshelves skimming books for a few minutes.`
+			`	"I found it," she says. "This is a modified version of the Symbol of Baphomet, the official symbol of an older incarnation of the Church of Satan. Contrary to popular belief at the time, they were never devil-worshipers of any sort. Instead, they believed in a sort of rationalist individualism and were one of the first organizations to fund large-scale settlements on Freedom. Since then, the symbol has been co-opted by miscreants.`
+			`	"But you didn't come here for a theological history disquisition. The letters around the pentagram have been replaced with galactic coordinates using an archaic notation." She smiles at you, "I wrote my Ph.D. thesis about that notation! Now I finally need that information for something other than getting a degree!`
```

Comment:
you forgot the " after degree!

Reply:
No, when the next paragraph is the same person continuing to speak it's convention to leave the closing mark off, because it's not actually closing the spoken part.

Reply:
I'm not sure that's the case in all of your conversations though

Reply:
Hopefully should be. I check that when proofreading, but certainly possible I missed one!

Reply:
Yeah I generally trust W1zrad but if you spot one that's been missed, obviously suggest a correction.